.github/workflows/sonar.yml

```
@@ -19,7 +30,7 @@ jobs:
           go-version: '1.24'
 
       - name: Generate Sonar Report
-        run: go test -coverpkg=./... -coverprofile=coverage.out -json ./... > sonar-report.json
+        run: go test -db_test_base="root@tcp(localhost:${{ job.services.mysql.ports[3306] }})/test?" -coverpkg=./... -coverprofile=coverage.out -json ./... > sonar-report.json
```

actionlint (1.7.4)

property access of object must be type of string but got "number"

(expression)